autofill/frame-parent.html

```diff
@@ -20,7 +21,7 @@ <h2>Cross origin iframe</h2>
 
   <style>
     iframe {
-      height: 2em;
+      height: 4em;
```

Comment:
This ensures that a) the tooltip sort of shows (it's not completely hidden) and b) it shows that it's cut off in extensions but not on macOS.

Reply:
Good idea :+1: